File: ticket_service/src/main/java/com/ticketflow/ticket_service/models/client_models/OrderModel.java
package com.ticketflow.ticket_service.models.client_models;

public class OrderModel {
    private Integer ticketId;
    private String buyerEmail;

    public OrderModel(Integer ticketId, String buyerEmail) {
        this.ticketId = ticketId;
        this.buyerEmail = buyerEmail;
    }

    public Integer getTicketId() {
        return ticketId;
    }

    public String getBuyerEmail() {
        return buyerEmail;
    }
}

File: backend/dotnet/ticketflow/TicketFlow.TicketService/Service/Models/OrderModel.cs
namespace TicketFlow.TicketService.Service.Models
{
    public class OrderModel
    {
        public OrderModel(int ticketId, string buyerEmail)
        {
            TicketId = ticketId;
            BuyerEmail = buyerEmail;
        }

        public int TicketId { get; }

        public string BuyerEmail { get; }
    }
}